tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -129,3 +137,23 @@ export const addUserToGroup = async ({
   })
   checkResponseStatus(response, 'Failed while adding an user to the group')
 }
+
+export const assignRole = async (admin: User, id: string, role: string): Promise<void> => {
+  const roleId = Roles[role]
+  if (!roleId) {
+    throw new Error(`unknown role "${role}"`)
+  }
+
+  const applicationEntities = await getAppEntities(admin)
+  const response = await request({
+    method: 'POST',
+    path: join('graph', 'v1.0', 'users', id, 'appRoleAssignments'),
+    user: admin,
+    body: JSON.stringify({
+      principalId: id,
+      appRoleId: roleId,
+      resourceId: applicationEntities[0].id
```

Comment:
use of index can be problematic later
```suggestion
      resourceId: applicationEntities[0].id
```